src/js/helpers/awardSummaryHelper.js

```diff
@@ -37,3 +37,8 @@ export const fetchAwardFederalAccounts = (data) => {
         }
     };
 };
+
+// The string ASST_AGG w/in a generatedUniqueId indicates an aggregated assistance type award
```

Comment:
Although I do not agree with having helpers and requests in the same file, we do handle helpers vaguely in the application and really should be addressed outside of this PR